platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/component/ComponentWiring.java

```diff
@@ -338,6 +350,118 @@ public <ELEMENT> OutputWire<ELEMENT> getSplitOutput() {
         return (OutputWire<ELEMENT>) splitterOutput;
     }
 
+    /**
+     * Get an output wire that will receive a specific type of routed data. Data routing is when a component has
```

Comment:
"that will receive a specific type of routed data"

I think it might be more clear to say "that will *emit* a specific type of routed data"

(if you decide to act on this, the comment also applies to `getSplitAndRoutedOutput`)

Reply:
changed